plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/form/types.ts

```diff
@@ -31,3 +37,35 @@ export interface AddressFormProps< T > {
 	// support inserting children at end of form
 	children?: React.ReactNode;
 }
+
+export interface AddressFieldsProps< T > extends AddressFormProps< T > {
+	// Overwriting the id for the fields.
+	id: string;
+	// Overwriting the address type for the fields.
+	addressType: FormType;
+	// Address 1 fields.
+	address1?: KeyedFormField;
+	// Address 2 fields.
+	address2?: KeyedFormField;
+	// Address form fields.
+	addressFormFields: AddressFormFields;
+	// Ref for fields.
+	fieldsRef: React.MutableRefObject<
```

Comment:
What is this for?

Reply:
Do you mean the interface in general or `fieldsRef` in specific?

Reply:
`fieldsRef` - I'm curious why we are passing a ref

Reply:
Good point. I've accidentally taken that over from `<ValidatedTextInput/>`, which I duplicated and moved into `<Address2Field/>`. Now that I look at it again, I realise that we do not need `fieldsRef` in `< Address2Field/>`. I just removed it accordingly. Thanks for catching this, @opr.